test/php/library/Icingadb/Util/PerfdataTest.php

```diff
@@ -399,4 +416,16 @@ public function testWhetherPercentagesAreHandledCorrectly()
             'Perfdata objects do not ignore impossible min/max combinations when returning percentages'
         );
     }
+
+    /**
+     * @depends testWhetherFromStringParsesAGivenStringCorrectly
+     */
+    public function testInvalidPerfDataHandling()
```

Comment:
You should add to cases where min is correct but max is not, and where min is incorrect but max is.